tests/testthat/test-ratio_calculations.R

```diff
@@ -271,5 +291,39 @@ describe("calculate_ratios", {
     expect_true(is.character(ratios_empty$PPORRESU))
     expect_true(is.character(ratios_empty$PPSTRESU))
   })
+ it("uses all alternative groups to ref_groups when test_groups is NULL", {
+    test_groups <- NULL
```

Comment:
my bad I think I miswrite in my suggestion this one

lintr: identation issue
```suggestion
  it("uses all alternative groups to ref_groups when test_groups is NULL", {
    test_groups <- NULL
```